awbw-notes.user.js

```diff
@@ -11,15 +11,26 @@
 
 (function() {
     'use strict';
-
     const key = "notes_" + gameId;
     var text = localStorage.getItem(key) || "";
 
     var textBox = document.createElement("textarea");
-    textBox.setAttribute("class", "playerNotes");
+    textBox.setAttribute("class", "notesClosable");
+    textBox.setAttribute("style", "position:inline;width:200px;height:100px;");
     textBox.value = text;
 
-    document.querySelector(".game-player-info").appendChild(textBox);
+    var closeButton = document.createElement("div");
+    closeButton.innerHTML = "&#9660;"
+    closeButton.setAttribute("id", "notesClose");
+
+    var divBox = document.createElement("div");
+    divBox.innerHTML = "<header class='notesClosable'>Notes:</header>";
```

Comment:
The transparent background here makes the text a little hard to read. Suggest a simple coloured background instead.

![image](https://user-images.githubusercontent.com/2514290/126900609-3eae498e-c0ea-40e0-8e0c-ac0bb55b371d.png)

Reply:
Fixed, good notes.